.github/workflows/shared-terraform-chatops.yml

```
@@ -21,869 +21,19 @@ defaults:
-            -owner "${{ needs.context.outputs.base_repo_owner }}"
-            -repo "${{ needs.context.outputs.base_repo_name }}"
-            -state ${{ steps.status.outputs.result }}
-            -context "test/terratest"
-            -description "Test run finished"
-            -url "https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }}"
-        env:
-          GITHUB_TOKEN: ${{ github.token }}
+    with:
+      pr_number: ${{ github.event.issue.number }}
+      runs-on: ${{ inputs.runs-on }}
+    
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)